config/smart_answers/check_benefits_financial_support_data.yml

```diff
@@ -185,7 +191,7 @@
         - wales
     - name: 15hrs_free_childcare_3_4yr_olds_england
       title: 15 hours of free childcare for 3 and 4-year-olds
-      description: You can get 570 hours of free childcare per year with an approved childcare provider if your child is 3 or 4 years old.
+      description: You can get 570 hours of free childcare per year with an approved childcare provider if your child is 3 or 4 years old and has not started school full-time.
```

Comment:
Just wondering, given we have HTML in this field sometimes as per the `<p>` examples above. 

Is it fine that this doesn't have a paragraph element wrapping? It seems surprising we mix both HTML and non-HTML in the same field - unless I'm missing something.

Reply:
You're right, we probably should be wrapping everything in `<p>` for consistency.
I believe the examples with a `<p>` are ones which require a line break between two text blocks, and ones without are single blocks.